jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -76,7 +98,7 @@ export const usePipeline = (
         }
       } catch (e) {
         if (e instanceof ServerConnection.ResponseError) {
-          setErrorMessage(e.message);
+          setErrorMessage('Error creating pipeline');
```

Comment:
Don't we want to return the server message here? (if we're passing a nicer error from the backend)

Reply:
This relates to the [this ticket](https://pachyderm.atlassian.net/browse/INT-925). I couldn't figure out how to actually get the message sent from the backend. I've been looking into this but I haven't had much luck yet. Nonetheless I think it should be addressed in future work (and hopefully before release)

Reply:
future work note ^